app/gen-server/lib/homedb/UsersManager.ts

```diff
@@ -582,6 +582,32 @@ export class UsersManager {
       .filter(fullProfile => fullProfile);
   }
 
+  /**
+   * Update users with passed property. Optional user properties that are missing will be reset to their default value.
+   */
+  public async overrideUser(userId: number, props: UserProfile): Promise<User> {
```

Comment:
I'm  a bit confused by the name of this function. Shouldn't it be called `updateUser` or `modifyUser`? A name like "override" to me suggests that we're temporarily getting a user that is different in some way from what's saved in the database, but that we are not saving this new value to the database.

Reply:
Right

Actually, this function is meant to replace all the user's properties in the DB (any property that is not passed in the `UserProfile` should be reset to their default value). And `update` or `modify` for me would imply the contrary, I would say (that's arguable).

What do you think?

Reply:
`resetUser`, `restoreUser`? Perhaps?

Reply:
Wait.. did you mean `overwriteUser` instead of `overrideUser`?

Reply:
Oops, you're right, I always confuse those two terms!

Reply:
Done